browser-test/src/question_lifecycle.test.ts

```diff
@@ -11,30 +11,41 @@ describe('normal question lifecycle', () => {
 
     const questions = await adminQuestions.addAllNonSingleBlockQuestionTypes('qlc-');
     const singleBlockQuestions = await adminQuestions.addAllSingleBlockQuestionTypes('qlc-');
+    const repeatedQuestion = 'qlc-repeated-number';
+    await adminQuestions.addNumberQuestion(
+      repeatedQuestion, 'description', '$this\'s favorite number', '', 'qlc-enumerator');
+    const allQuestions = questions.concat(singleBlockQuestions);

```

Comment:
nit (non-blocking): Could we move this line right below the single block question creation? It confused me that we create the repeated question but don't add it until later

Reply:
I added some white space and a comment about what `allQuestions` is.